### PR TITLE
fixed broken items credits to walshy

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

fixed broken items credits to walshy
## Changes

fixed broken items credits to walshy
## Related Issues

#1187 
#1184 
## Testability

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java [hunk 1]
```diff
@@ -42,7 +42,10 @@ public int getOutput() {
 		fortune++;
 		return fortune;
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/EnderBackpack.java [hunk 2]
```diff
@@ -16,7 +16,10 @@
 	public EnderBackpack(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/EnderBackpack.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/GoldPan.java [hunk 3]
```diff
@@ -46,7 +46,10 @@ public void postRegister() {
 		chanceClay = (int) Slimefun.getItemValue(getID(), "chance.CLAY");
 		chanceFlint = (int) Slimefun.getItemValue(getID(), "chance.FLINT");
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/GoldPan.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/GrapplingHook.java [hunk 4]
```diff
@@ -30,7 +30,10 @@
 	public GrapplingHook(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, String[] keys, Object[] values) {
 		super(category, item, recipeType, recipe, keys, values);
 	}
-
+    @Override
+    protected boolean areItemHandlersPrivate() {
+        return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/GrapplingHook.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/InfernalBonemeal.java [hunk 5]
```diff
@@ -18,7 +18,10 @@
 	public InfernalBonemeal(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
 		super(category, item, id, recipeType, recipe, recipeOutput);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/InfernalBonemeal.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java [hunk 6]
```diff
@@ -18,7 +18,10 @@
 import me.mrCookieSlime.Slimefun.Objects.handlers.BlockBreakHandler;
 import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
-
+@Override
+protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java]: What on earth is this formatting?
Also please move that method further down.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicEyeOfEnder.java [hunk 7]
```diff
@@ -17,7 +17,10 @@
 	public MagicEyeOfEnder(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicEyeOfEnder.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/KnowledgeTome.java [hunk 8]
```diff
@@ -25,7 +25,10 @@
 	public KnowledgeTome(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/KnowledgeTome.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/KnowledgeFlask.java [hunk 9]
```diff
@@ -17,7 +17,10 @@
 	public KnowledgeFlask(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
 		super(category, item, recipeType, recipe, recipeOutput);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/KnowledgeFlask.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicSugar.java [hunk 10]
```diff
@@ -21,7 +21,10 @@
 	public MagicSugar(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, String[] keys, Object[] values) {
 		super(category, item, recipeType, recipe, keys, values);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicSugar.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PickaxeOfTheSeeker.java [hunk 11]
```diff
@@ -18,7 +18,10 @@
 	public PickaxeOfTheSeeker(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PickaxeOfTheSeeker.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableCrafter.java [hunk 12]
```diff
@@ -16,7 +16,10 @@
 	public PortableCrafter(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableCrafter.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableDustbin.java [hunk 13]
```diff
@@ -18,7 +18,11 @@
 	public PortableDustbin(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableDustbin.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableGEOScanner.java [hunk 14]
```diff
@@ -14,7 +14,10 @@
 	public PortableGEOScanner(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, id, recipeType, recipe);
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/PortableGEOScanner.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/SeismicAxe.java [hunk 15]
```diff
@@ -30,7 +30,10 @@
 	public SeismicAxe(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/SeismicAxe.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/StormStaff.java [hunk 16]
```diff
@@ -45,7 +45,10 @@ private static ItemStack getCraftedOutput() {
 		item.setItemMeta(im);
 		return item;
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/StormStaff.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/TelepositionScroll.java [hunk 17]
```diff
@@ -17,7 +17,10 @@
 	public TelepositionScroll(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/TelepositionScroll.java]: This needs to be removed.

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/WindStaff.java [hunk 18]
```diff
@@ -21,7 +21,10 @@
 	public WindStaff(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, recipeType, recipe);
 	}
-	
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/WindStaff.java]: This needs to be removed.

**Comment by WalshyDev** [comment]
There once was a Panda, he made a commit. It got rejected and he deleted in rage! Next he decided to make 20 commits rather than 1 because he hates our emails

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java [hunk 19]
```diff
@@ -1,74 +0,0 @@
-package me.mrCookieSlime.Slimefun.Objects.SlimefunItem;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java]: You should not delete the Enhanced Furnace

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java [hunk 20]
```diff
@@ -19,6 +19,11 @@
 import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 
+@Override
+protected boolean areItemHandlersPrivate() {
+	return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java]: Please update the formatting on this and move the method further down, it shouldn't be at the top.

**Comment by J3fftw1** [comment]
fixed requested changes

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java (hunk 19 above)
Comment (J3fftw1) [reply src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java]: added it back

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java (hunk 20 above)
Comment (J3fftw1) [reply src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java]: changed it

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/gps/GPSTransmitter.java [hunk 21]
```diff
@@ -41,11 +41,14 @@ public boolean onBreak(Player p, Block b, SlimefunItem item, UnregisterReason re
 	
 	public abstract double getMultiplier(int y);
 	public abstract int getEnergyConsumption();
-
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/gps/GPSTransmitter.java]: This should be removed.

Comment (J3fftw1) [reply src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/electric/gps/GPSTransmitter.java]: fixed

**Comment by TheBusyBiscuit** [comment]
@J3fftw1 BTW for future reference:
Use the syntax "Fixes #1187" and "Fixes #1184" to automatically close the related issues when merging.

**Pull request merged**